distrobox-create

```diff
@@ -118,9 +124,13 @@ if [ -z "${container_clone}" ] && [ -z "${container_image}" ]; then
 	container_image="${container_image_default}"
 fi
 
+if [ "${container_name}" == "" ]; then
+	container_name=$(basename $container_image | sed -E 's/:/_/g')
```

Comment:
I like this idea :smile: what about using an hyphen instead of underscore? so we can build the default name we had before: `fedora-toolbox-35` from `fedora-toolbox:35`

Reply:
Also there are a couple of fixes about quoting stuff, to put it in line with the rest of the POSIX stile:

`container_name="$(basename "${container_image}" | sed -E 's/:/-/g')"`